VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -61,7 +61,7 @@
     private RadioGroup discussionQuestionSegmentedGroup;
 
     @InjectView(R.id.topics_spinner)
-    private Spinner topicsSpinner;
+    private AppCompatSpinner topicsSpinner;
```

Comment:
`AppCompatSpinner` extends `Spinner` (and we were even using `AppCompatSpinner` previously), so this change wasn't necessary, and in fact is counterproductive because we don't want to fix the type to a specific implementation.